components/explorer/src/lib.rs

```diff
@@ -372,6 +372,10 @@ impl<'a> IntoIterator for &'a BlockWithTransactions {
 /// | `location` | [`TxLocation`] | Location of the transaction in the block |
 /// | `location_proof` | [`ListProof`]`<`[`Hash`]`>` | Proof of transaction inclusion into a block |
 /// | `status` | (custom; see below) | Execution status |
+/// | `time` | [`DateTime`]`<`[`Utc`]`>` | Commitment time* |
+/// \* By commitment time we mean an approximate commitment time of the block
+/// which includes the transaction. This time is calculated as the arithmetic average
```

Comment:
It used to be the median time — has that changed?

Reply:
Yeah. Good catch. It's still median time 😄